### DISCUSSION TITLE
如何设置软件随系统一起启动？

### DISCUSSION THREAD

**Discussion by 123456ds** [jeessy2/ddns-go#1108] [Q&A] [answered]
使用的是window系统，有用户密码验证。
为该软件添加了计划任务&启动项目，但只有进入用户桌面才会启动。
想让该软件在用户密码验证页面就加载。
![%E4%B8%8B%E8%BD%BD.jpg](https://p.sda1.dev/17/0853dd457a55c050de5f17fd9a54434f/下载.jpg)

**Comment by jeessy2** [comment, marked as answer]
可以安装为服务

**Reply by 123456ds** [reply]
有使用cmd命令【.\ddns-go.exe -s install】安装服务
![%E5%B1%8F%E5%B9%95%E6%88%AA%E5%9B%BE%202024-05-11%20104758.png](https://p.sda1.dev/17/c8022aba1330baca6a8930106d363613/屏幕截图 2024-05-11 104758.png)

**Reply by jeessy2** [reply]
设置登录凭证？这些自己研究下吧。
